nailgun/entities.py

```diff
@@ -1022,6 +1022,24 @@ def content_reclaim_space(self, synchronous=True, timeout=None, **kwargs):
         response = client.post(self.path('content_reclaim_space'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
+    def content_verify_checksum(self, synchronous=True, timeout=None, **kwargs):
+        """Check for missing or corrupted artifacts, and attempt to redownload them.
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
```

Comment:
As per my understanding about synchronous execution, 
if `synchronous=True` then don't wait for current task to execute completely and go for next step
if `synchronous=False` then wait till current execution to finish completely.

Please correct me if I am wrong.
```suggestion
        :param synchronous: What should happen if the server returns an HTTP
            202 (accepted) status code?  Immediately return the server's response 
 if synchronous is ``True``. Wait for the task to complete otherwise.
```

Reply:
IMHO it's exaclty the opposite - synchronous means we wait until the task is finished.

Reply:
Okay, I see every nailgun entities define doc string in same way.